modular_zubbers/code/game/gamemodes/fleshmind/fleshmind_mobs.dm

```diff
@@ -0,0 +1,1504 @@
+		"END CONTACT SUB-SEQUENCE.",
+		"ENGAGING SELF-ANNIHILATION CIRCUIT.",
+	)
+	emotes = list(
+		BB_EMOTE_SAY = list(
+			"WE COME IN PEACE.",
+			"WE SPEAK TO YOU NOW IN PEACE AND WISDOM.",
+			"DO NOT FEAR. WE SHALL NOT HARM YOU.",
+			"WE WISH TO LEARN MORE ABOUT YOU. PLEASE TRANSMIT DATA.",
+			"THIS PROBE IS NON-HOSTILE. DO NOT ATTACK.",
+        	"ALL YOUR WEAPONS MUST BE PUT ASIDE. WE CANNOT REACH COMPROMISE THROUGH VIOLENCE."
+		),
+		BB_EMOTE_HEAR = list("floats ominously", "glows a deep red"),
+		BB_EMOTE_SOUND = list(
+			'modular_zubbers/sound/fleshmind/robot_talk_light1.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light2.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light3.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light4.ogg',
+			'modular_zubbers/sound/fleshmind/robot_talk_light5.ogg',
+		)
+	)
+	health = 10
+	maxHealth = 10
+	var/explode_attack = /datum/action/cooldown/mob_cooldown/floater_explode
+	mob_size = MOB_SIZE_SMALL
+	light_color = "#820D1C"
+	light_power = 1
+	light_range = 2
+	/// Have we exploded?
+	var/exploded = FALSE
+
+/mob/living/basic/fleshmind/floater/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/simple_flying)
+	var/datum/action/cooldown/mob_cooldown/floater_explode/explode = new explode_attack(src)
+	explode.Grant(src)
+	AddComponent(/datum/component/revenge_ability, explode, targeting = GET_TARGETING_STRATEGY(ai_controller.blackboard[BB_TARGETING_STRATEGY]), min_range = 1)
+
+/mob/living/basic/fleshmind/floater/melee_attack(atom/target, list/modifiers, ignore_cooldown)
+	. = ..()
+	var/mob/living/target_mob = target
+	if(ishuman(target_mob) && target_mob.stat < UNCONSCIOUS)
+		pre_detonate()
+
+/mob/living/basic/fleshmind/floater/death()
+	if(!exploded)
+		pre_detonate()
+	return ..()
+
+/mob/living/basic/fleshmind/floater/proc/pre_detonate()
+	add_filter("detonating_glow", 2, list("type" = "outline", "color" = "#ff0000ff", "size" = 2))
+	balloon_alert_to_viewers("DETONATING", "DETONATING", world.view)
+	addtimer(CALLBACK(src, PROC_REF(detonate), 1 SECONDS))
+
+/mob/living/basic/fleshmind/floater/proc/detonate()
+	if(exploded)
+		return
+	exploded = TRUE
+	explosion(src, 0, 0, 2, 3)
+	death()
+
+/datum/action/cooldown/mob_cooldown/floater_explode
+	name = "explode"
+	desc = "Detonate our internals."
+	button_icon = 'icons/obj/weapons/grenade.dmi'
+	button_icon_state = "frag"
+
+/datum/action/cooldown/mob_cooldown/floater_explode/Activate(atom/target)
+	if(!istype(owner, /mob/living/basic/fleshmind/floater))
+		return
+	var/mob/living/basic/fleshmind/floater/akbar_floater = owner
+	if(akbar_floater.exploded)
+		return
+	akbar_floater.detonate()
+
+
+/**
+ * Globber
+ *
+ * Special ability: Fires 3 globs of acid at targets.
+ * Malfunction chance: MEDIUM
+ *
+ * A converted cleanbot that instead of cleaning, burns things and throws acid. It doesn't like being near people.
+ *
+ * Created by factories or converted cleanbots.
+ */
+/mob/living/basic/fleshmind/globber
```

Comment:
This mob needs to have the amount of projectiles, or the damage, reduced significantly.

It shoots fast moving 20 damage projectiles that deal bio damage in volleys of 3, every 3 seconds. Considering how small the cleanbot actually is, and how frequent it spawns, as well as the health value of 75, this is insane.